README.md

```diff
@@ -1,2 +1,570 @@
-# predict-credit-risk-with-jupyter-on-cloud-pak-for-data
-Use IBM Cloud Pak for Data and Jupyter to predict credit risk
+# Predict Credit Risk With Jupyter on Cloud Pak for data
+
+In this Code Pattern, we will go through the process of exploring a data set and building a predictive model that can be used to determine the likelyhood of a credit loan having 'Risk' or 'No Risk'. For this use case, the machine learning model we are building is a classification model that will return a prediction of 'Risk' (the features of the loan applicant predict that there is a good chance of default on the loan) or No Risk (the applicant's inputs predict that the loan will be paid off). The approach we will take in this lab is to some fairly popular libraries / frameworks to build the model in Python using a Jupyter notebook. Once we have built the model, we will make it available for deployment so that it can be used by others.
```

Comment:
- likelihood
- last 'No Risk' should have quotes to be consistent
- "in this lab is to some". Missing verb like "use" or "employ"

Reply:
fixed, thanks.